pyEpiabm/pyEpiabm/core/person.py

```diff
@@ -98,14 +98,21 @@ def update_time_to_status_change(self) -> None:
         new_time = random.randint(1, 10)
         self.time_of_status_change = new_time
 
-    def add_place(self, place):
+    def add_place(self, place, person_group: int = 0):
         """Method adds a place to the place list if the person visits
-        or is associated with this place.
+        or is associated with this place. Places are saved as a tuple
+        with the place as the first entry and group person is associated
```

Comment:
"and _the_ group _a_ person is associated with" perhaps? Not sure I understand this docstring